Contributing-code-to-Oppia.md

```diff
@@ -76,7 +76,7 @@ We also **strongly recommend** looking through the resources under "Developing O
 
 Welcome! Please make sure to follow the instructions above if you haven't already.
 
-After that, you can choose a good first issue from the [list of good first issues](https://github.com/oppia/oppia/labels/good%20first%20issue). These issues are hand-picked to ensure that you don't run into unexpected roadblocks while working on them, and each of them should have clear instructions for new contributors. If you see one that doesn't, please let us know via [GitHub Discussions](https://github.com/oppia/oppia/discussions) and we'll fix it. For other issues, you might need to be more independent because we might not know how to solve them either.
+After that, you can choose a good first issue from the [list of unassigned good first issues](https://github.com/oppia/oppia/issues?q=is%3Aopen+label%3A%22good+first+issue%22+no%3Aassignee). These issues are hand-picked to ensure that you don't run into unexpected roadblocks while working on them, and each of them should have clear instructions for new contributors. If you see one that doesn't, please let us know via [GitHub Discussions](https://github.com/oppia/oppia/discussions) and we'll fix it. For other issues, you might need to be more independent because we might not know how to solve them either.
```

Comment:
Providing the link to unassigned issues is a good idea. However, I had a thought in mind: there are often cases where a new contributor gets assigned to an issue but then disappears. For e.gh: https://github.com/oppia/oppia/issues/20564.

So, I think it's fine to show such issues to them.

Just wanted to check this thought with you!

Reply:
Hi @Ash-2k3, that's a great catch, thanks!

I think this is not the place to fix that, though. We already have multiple cases where new contributors try to claim issues that have already been assigned. Instead, we should have some way to check for stale assignments and drop those. I will file an issue to get oppiabot to do that.

Reply:
Filed https://github.com/oppia/oppia/issues/21294

Reply:
Looks good, thanks!